src/mapper/postTransformations.ts

```diff
@@ -0,0 +1,129 @@
+import cytoscape, { ElementDefinition, Position } from 'cytoscape';
+import { NodeType } from '../models/nodeType';
+import { getSymbol, NodeSymbol, SymbolRotation } from '../symbol-api';
+import { arrayEquals } from '../utils/arrayEquals';
+import { mergeElementsByKey } from './mergeElements';
+import { postProcessSvgTag } from './transformations';
+
+export type Transformation = {
+	transformNew: (element: ElementDefinition) => ElementDefinition[];
+	transformUpdate: (element: ElementDefinition, cy: cytoscape.Core) => void;
+};
+
+export const createSvgTransformation = (iconNode2Connectors: { [iconNode: string]: ElementDefinition[] }): Transformation => {
+	const transform = (element: ElementDefinition) => {
+		const rotation = parseInt(element.data.rotation) as SymbolRotation;
+		const symbol = getSymbol(element.data.symbolId, { rotation: rotation });
+
+		const parentNode: ElementDefinition = {
+			data: {
+				...element.data,
+				imageHeight: `${symbol.height}px`,
+				imageWidth: `${symbol.width}px`,
+				[postProcessSvgTag]: false,
+			},
+		};
+
+		const children = iconNode2Connectors[element.data.id!];
+
+		const connectorElements = children
+			.filter(({ data }) => data.nodeType === NodeType.SymbolConnector)
+			.map(({ data }) => {
+				return {
+					data: data,
```

Comment:
Can be just `data`

Reply:
Learning so many cool javascript tricks from your reviews 😄